homeassistant/components/roborock/__init__.py

```diff
@@ -35,9 +35,17 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     try:
         home_data = await api_client.get_home_data(user_data)
     except RoborockInvalidCredentials as err:
-        raise ConfigEntryAuthFailed("Invalid credentials.") from err
+        raise ConfigEntryAuthFailed(
```

Comment:
Is config entry exceptions user-facing?

Reply:
I believe so. If I am not mistaken, it shows up on top of the exclamation mark on the integration page when it is unavailable.

Reply:
Is that the exception or the translated message?

Reply:
Seemingly just the exception message - not the translated message

Reply:
So we remove it then?

Reply:
Yeah I can - does seem a bit strange though, you would think that is one of the main exceptions you would want translated because it pops up on users dashboard

Reply:
Agree so that's why I initially asked if this was actually translated or not.
Let's see it gets added.

Reply:
Well does it make sense for me to leave it in so that if/when it gets added it works? Since having it isn't breaking?

Reply:
I think we shouldn't merge dead code but I guess this is a bit borderline and we should try to get this translated as well.

Reply:
I put in a PR to make translation possible for Config entry errors but in the meantime I changed my mind so let's put it back so it's there when support gets in.

Reply:
Sorry for the delay in doing this. but it should be done now.